scripts/packages/chocolatey/tools/chocolateyinstall.ps1

```diff
@@ -15,19 +15,20 @@ $packageDir = Split-Path -parent $toolsDir
 Install-ChocolateyZipPackage -PackageName "$packageName" `
   -Url64bit "$($paramsText[0])" `
   -Checksum64 "$($paramsText[1])" `
-  -Checksum64Type "sha256" `
+  -ChecksumType64 "sha256" `
   -UnzipLocation "$packageDir"
 
 write-host "Ensure that msys2 dll is present in PATH to allow bazel to be run from non-msys2 shells"
 $pp = Get-PackageParameters $env:chocolateyPackageParameters
-Install-ChocolateyPath -PathToInstall "$($pp.msys2Path)\usr\bin" -PathType "Machine"
+$msys2Path = ($pp.msys2Path, "c:\tools\msys64" -ne $null)[0]
```

Comment:
That's idiomatic PowerShell null-coalescing, that is. Expressive, huh? :cry:

"Take the first item from the array that is not equal to null".